src/Umbraco.Web.UI.Client/src/views/membertypes/create.html

```diff
@@ -1,47 +1,53 @@
+                <li class="umb-action">
+                    <button type="button" class="umb-action-link umb-outline btn-reset" ng-click="showCreateFolder()">
+                        <i class="large icon icon-folder" aria-hidden="true"></i>
+                        <span class="menu-label">
+                            <localize key="general_folder"></localize>
+                        </span>
+                    </button>
+                </li>
+                <li class="umb-action">
+                    <button type="button" class="umb-action-link umb-outline btn-reset" ng-click="createMemberType()">
+
+                        <i class="large icon icon-item-arrangement"></i>
+
+                        <span class="menu-label">
+                            <localize key="general_new">New</localize>&nbsp;
+                            <localize key="content_memberType">Member type</localize>
+                        </span>
+
+                    </button>
+                </li>
+            </ul>
+        </div>
+
+        <div class="umb-pane" ng-if="model.creatingFolder">
+            <form novalidate name="createFolderForm" ng-submit="createFolder()" val-form-manager>
+
+                <umb-control-group label="Enter a folder name" hide-label="false">
+                    <input type="text" name="folderName" ng-model="model.folderName"
+                        class="umb-textstring textstring input-block-level" required />
+                </umb-control-group>
+
+                <button type="submit" class="btn btn-primary">
+                    <localize key="general_create">Create</localize>
                 </button>
-            </li>
-            <li class="umb-action">
-                <button type="button" class="umb-action-link umb-outline btn-reset" ng-click="createMemberType()">
-
-                    <i class="large icon icon-item-arrangement"></i>
+            </form>
+        </div>
 
-                    <span class="menu-label">
-                        <localize key="general_new">New</localize>&nbsp;
-                        <localize key="content_memberType">Member type</localize>
-                    </span>
-
-                </button>
-            </li>
-        </ul>
     </div>
 
-    <div class="umb-pane" ng-if="model.creatingFolder">
-        <form novalidate name="createFolderForm"
-              ng-submit="createFolder()"
-              val-form-manager>
-
-            <umb-control-group label="Enter a folder name" hide-label="false">
-                <input type="text" name="folderName" ng-model="model.folderName" class="umb-textstring textstring input-block-level" required />
-            </umb-control-group>
-
-            <button type="submit" class="btn btn-primary"><localize key="general_create">Create</localize></button>
-        </form>
+    <div class="umb-dialog-footer btn-toolbar umb-btn-toolbar" ng-if="!model.creatingFolder">
+        <button class="btn btn-info" ng-click="nav.hideDialog(true)">
```

Comment:
@nathanwoulfe I think it would be great to add `type` attribute here as well like here:
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/views/datatypes/create.html#L43

But not sure it is added in all dialogs.

Reply:
@bjarnef I've updated to add type attribute in doctype, media and member dialogs 👍